src/templates/static/legal/tac_sections/tac.jsx

```diff
@@ -35,11 +35,19 @@ const Tac = () => (
-        <p>{it.L('For Deriv Investments (Europe) Limited and Deriv (Europe) Limited, the applicable Jurisdiction shall be Malta.')}</p>
-        <p>{it.L('For Deriv (MX) Ltd, the applicable Jurisdiction shall be the Isle of Man.')}</p>
-        <p data-show='-eucountry'>{it.L('For Deriv (SVG) LLC, the applicable Jurisdiction shall be St. Vincent and the Grenadines.')}</p>
+        <p>{it.L('It is the responsibility of clients accessing this site to make sure that they understand and fully follow the relevant laws and regulations of their own country. The Company does not support any offerings or make any solicitations that are illegal in a client\'s country, and clients cannot use their account for such prohibited offerings. In some countries, the Company may not be permitted to offer products and services to clients unless they have taken proactive steps to solicit information about the Company\'s products and services and requested to be provided with them. Access to this site and the offering of certain trades via this site may be also restricted in some jurisdictions. It is the client\'s responsibility to know about the restrictions that apply in the client\'s country of residence and to observe them, as in continuing to access the Company\'s products and services, the client represents to the Company that the client is doing so legally and that, where relevant, the client has solicited information about the Company\'s products and services and requested to be provided with them.')}</p>
+        <p>{it.L('Depending on where the client lives, the client\'s account will be governed by the laws of the country of incorporation of the company that the client\'s account is registered with as set out below. If the client has registered more than one account with more than one company, each account will be governed by the law of the country of incorporation of the company that each individual account has been registered with. The client agrees to submit to the applicable jurisdiction\'s courts and bring any claim the client may have before them. These courts will have sole authority to settle any disputes that may arise between the client and the Company.')}</p>
+        <p>{it.L('If a court or regulatory body of competent jurisdiction declares any provision of the Terms to be invalid or unenforceable, the rest of the Terms shall remain in full force and effect.')}</p>
+        <p>{it.L('The applicable jurisdiction shall be one of the following:')}</p>
+        <ul className='bullet'>
+            <li>{it.L('Deriv Investments (Europe) Limited and Deriv (Europe) Limited: Malta')}</li>
+            <li>{it.L('Deriv (MX) Ltd: the Isle of Man')}</li>
+            <li>{it.L('Deriv (FX) Ltd: Labuan, Territory of Malaysia')}</li>
+            <li>{it.L('Deriv (BVI) Ltd: British Virgin Islands')}</li>
+            <li>{it.L('Deriv (V) Ltd: Vanuatu')}</li>
+            <li>{it.L('Deriv (SVG) LLC: St. Vincent and the Grenadines')}</li>
```

Comment:
we used to only show this for non-eu countries, but now all of these are showing to everyone. is that intended? can you check with compliance?

Reply:
Yes, sure. I’m checking

Reply:
The `Governing laws and jurisdiction` section should be visible to all users.